tests/FileStorageTest.php

```diff
@@ -14,25 +14,264 @@
 class FileStorageTest extends ClientTestCase
```

Comment:
Pletu se, nebo se vytvořené filestorage nikdy nemažou? (tj. zustávají zaregistrované v tom cloudu)

Reply:
ty filestorage si ty v cloudu vytvoříš ručně. Tohle jenom zaregistruje do KBC, aby o nich věděla. 

Mazat pak ty záznamy u tebe v DB, je trochu tricky, protože by sis smazal všechny zaregistrované storage:) Stejně jako se nemažou tokeny, projekty ... To je podle mě trochu bolístka manage testů, které by měli běžet na oddělené DB.